docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+services:
+  pg:
+    image: timescale/timescaledb:latest-pg13
+    restart: unless-stopped
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - ./pg-init:/docker-entrypoint-initdb.d
+      - pgdata:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
+  
+  grafana:
+    image: grafana/grafana:8.5.3
+    restart: unless-stopped
+    ports:
+      - 3000:3000
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+      - GF_USERS_ALLOW_SIGN_UP=false
+
+      - GF_DATABASE_TYPE=postgres
```

Comment:
using existing existing postgres db for storing grafana's users/dashboards/config/etc. sqlite would've been used by default

Reply:
default db name is `grafana`